starknet_devnet/general_config.py

```diff
@@ -34,7 +33,6 @@ def build_devnet_general_config(chain_id: StarknetChainId):
                 "chain_id": chain_id.value,
                 "fee_token_address": hex(FeeToken.ADDRESS),
             },
-            "tx_version": SUPPORTED_TX_VERSION,
```

Comment:
is tx_version now specified elsewhere?

Reply:
I'll check but it was removed from `StarknetGeneralConfig` - that's for sure

Reply:
it's ok, I guess it's not really important since specifying the supported version wasn't something that users could modify.